src/cfg/cfg-traversal.h

```diff
@@ -88,9 +96,12 @@ struct CFGWalker : public ControlFlowWalker<SubType, VisitorType> {
 
   // stack of the last blocks of try bodies
   std::vector<BasicBlock*> tryStack;
-  // stack of the first blocks of catches that throwing instructions should
-  // unwind to at any moment. Because there can be multiple catch blocks for a
-  // try, we maintain a vector of first blocks of catches.
+  // Stack of the blocks that can reach the first blocks of catches that
+  // throwing instructions should unwind to at any moment. That is, the topmost
+  // item in this vector relates to the current try-catch scope, and the vector
+  // there is a list of the items that can reach catch blocks (each item is
+  // assumed to be able to reach any of the catches, although that could be
+  // improved perhaps).
```

Comment:
How about changing the name to `unwindingInstsStack` or `throwingInstsStack` or something? This doesn't contain catch blocks anymore.